### PR TITLE
Make phase walls don't reflect lasers

### PR DESCRIPTION
**Pull request by GlennFolker**
I noticed that phase walls sometimes reflect `LaserBulletType` and/or `ContinuousLaserBulletType`

### CODE REVIEW

**Comment by ThePythonGuy3** [comment]
Nice pr name, perfectly descriptive

**Comment by GlennFolker** [comment]
a

**Comment by GlennFolker** [comment]
Done. Title has been changed ***two times***.

**Comment by DeltaNedas** [comment]
meetaglis wall 10m hp and refracts/deflects lasers

File: annotations/src/main/resources/revisions/EffectState/6.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{version:6,fields:[{name:color,type:arc.graphics.Color,size:-1},{name:data,type:java.lang.Object,size:-1},{name:effect,type:mindustry.entities.Effect,size:-1},{name:lifetime,type:float,size:4},{name:offsetX,type:float,size:4},{name:offsetY,type:float,size:4},{name:parent,type:mindustry.gen.Posc,size:-1},{name:rotation,type:float,size:4},{name:time,type:float,size:4},{name:x,type:float,size:4},{name:y,type:float,size:4}]}
```
Comment (SummetDev) [inline annotations/src/main/resources/revisions/EffectState/6.json:1]: You shouldn't commit this file.

**Comment by grimsoul10** [comment]
How Glenn how?

**Comment by Anuken** [comment]
I'll do this myself.

File: annotations/src/main/resources/revisions/EffectState/6.json (hunk 1 above)
Comment (GlennFolker) [reply annotations/src/main/resources/revisions/EffectState/6.json:1]: Sorry; I just added the 2 lines and this file showed up after building Mindustry. I have no idea why.

Comment (MEEPofFaith) [reply annotations/src/main/resources/revisions/EffectState/6.json:1]: Same happens to me, I just ignore it.

**Pull request closed without merging**